docs/compatibility.md

```diff
@@ -357,6 +357,21 @@ Also parsing of some values will not produce bit for bit identical results to wh
 They are within round-off errors except when they are close enough to overflow to Inf or -Inf which
 then results in a number being returned when the CPU would have returned null.
 
+### CSV ANSI day time interval
+Apache Spark has an overflow issue when reading ANSI day time interval, the plugin fixed it on GPU.
```

Comment:
Can we use "the RAPIDS Accelerator" instead of "the plugin"?

i.e. 

```
Apache Spark can [overflow](https://issues.apache.org/jira/browse/SPARK-38520) when reading ANSI 
day time interval values. The RAPIDS Accelerator does not overflow and as such is not bug for bug 
compatible with Spark in this case.
```

Reply:
done